server/api/views/uploadFile/title.py

```diff
@@ -0,0 +1,56 @@
+import re
+
+import fitz
+
+from server.api.services.openai_services import openAIServices
```

Comment:
I got a ModuleNotError here when starting the Docker containers and fixed it by changing this to a relative import:

`from ...services.openai_services import openAIServices`